content/sensu-go/5.16/reference/hooks.md

```diff
@@ -19,9 +19,9 @@ menu:
   - [Capture the process tree](#capture-the-process-tree)
   - [Check hook using token substitution](#check-hook-using-token-substitution)
 
-Hooks are executed in response to the result of a check command execution and based on the exit status code of that command (ex: `1`).
-Hook commands can also receive JSON serialized Sensu client data via `STDIN`.
-You can create, manage, and reuse hooks independently of checks.
+Hooks are commands the agent executes in response to a check result before creating a monitoring event. 
+Hooks create context-rich events by gathering related information based on the exit status code of a check (ex: `1`).
+Hook commands can also receive JSON serialized Sensu client data via `STDIN`, and you can create, manage, and reuse hooks independently of checks.
```

Comment:
I think this is burying the lede a bit. IMO reusability of hooks (an operator feature) is likely a bigger selling point than the capability for hook plugins to receive event input (a plugin developer feature).

Reply:
I moved the reusability point into the second sentence and added "reusable" to the first sentence. What do you think?

I will also change "related" to "relevant" in the glossary definition for hooks.